src/views/DocumentList.vue

```diff
@@ -103,9 +103,10 @@ import SecurityPoliciesEditor from '@/doc-editors/SecurityPoliciesEditor.vue'
 import RateLimitsEditor from '@/doc-editors/RateLimitsEditor.vue'
 import GlobalFilterListEditor from '@/doc-editors/GlobalFilterListEditor.vue'
 import FlowControlPolicyEditor from '@/doc-editors/FlowControlPolicyEditor.vue'
+import CloudFunctionsEditor from '@/doc-editors/CloudFunctionsEditor.vue'
 import GitHistory from '@/components/GitHistory.vue'
 import {defineComponent, shallowRef} from 'vue'
-import {ColumnOptions, Commit, Document, DocumentType, GenericObject} from '@/types'
+import {ColumnOptions, Commit, Document, DocumentType, GenericObject, CloudFunctionsPhase} from '@/types'
```

Comment:
Keep types in alphabetical order

```suggestion
import {CloudFunctionsPhase, ColumnOptions, Commit, Document, DocumentType, GenericObject} from '@/types'
```